src/components/Breadcrumb/Breadcrumb.ts

```diff
@@ -0,0 +1,234 @@
+// Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE in the project root for license information.
+
+/**
+ * @namespace fabric
+ */
+namespace fabric {
+  "use strict";
+
+  /**
+   * Breadcrumb component
+   *
+   * Shows the user"s current location in a hierarchy and provides a means of navigating upward.
+   *
+   */
+  export class Breadcrumb {
+    // medium breakpoint
+    private static MEDIUM: number = 639;
```

Comment:
Actually this would need to be outside of the class implementation to work as a constant which creates other issues. So we can leave this as is.